sql/2024/performance/bfcache_unload.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TEMPORARY FUNCTION getUnloadHandler(audit STRING)
+RETURNS BOOL LANGUAGE js AS '''
+try {
+  var $ = JSON.parse(audit);
+  return $.details?.items?.some(n => n.value?.toLowerCase() === "unloadhandler");
+} catch (e) {
+  return false;
+}
+''';
+
+WITH lh AS (
+  SELECT
+    client,
+    page,
+    rank,
+    getUnloadHandler(JSON_EXTRACT(lighthouse, '$.audits.deprecations')) AS has_unload
+  FROM
+    `httparchive.all.pages`
+  WHERE
+    date = '2024-06-01' AND
+    is_root_page
```

Comment:
So for unload you've used home pages, but for CCNS you've used all pages. Should they use the same methodology?

Reply:
Agreed and updated.